docs/atoz.md

```diff
@@ -0,0 +1,487 @@
+
+~~~bash
+chmod 700 ~/.ssl
+chmod 644 ~/.ssl/wetty.crt 
+chmod 600 ~/.ssl/wetty.key
+~~~
+
+This is all we need to do for now in regards to https.
+
+### Generate the ssh key file
+
+**Why?** So that later we can set up automatic login via `ssh`. Our instance will authorise using this key file stored locally. 
+
+Make the required directory, if it does not exist, using this command:
+
+~~~bash
+mkdir -p ~/.ssh
+~~~
+
+Create the `ssh` private key using `ed25519` that we need to authorise our local connection, using this command:
+
+~~~bash
+ssh-keygen -q -C "wetty-keyfile" -t ed25519 -N '' -f ~/.ssh/wetty 2>/dev/null <<< y >/dev/null
+~~~
+
+**Important Note:** You must add the public key to your `authorized_keys` file in order to be able to log in using your `ssh` key file when accessing `wetty` via a web browser.
+
+Copy the key to our `~/.ssh/authorized_keys` file, using this command:
+
+~~~bash
+cat ~/.ssh/wetty.pub >> ~/.ssh/authorized_keys
+~~~
+
+Now give these file and folders the correct permissions, using these commands:
+
+~~~bash
+chmod 700 ~/.ssh
+chmod 644 ~/.ssh/authorized_keys
+chmod 600 ~/.ssh/wetty
+~~~
+
+**Optional:** A housekeeping command. If you need to remove all entries of the `wetty` public key with the comment `wetty-keyfile` from the `~/.ssh/authorized_keys` file use this command. Otherwise ignore this.
+
+~~~bash
+sed -r '/^ssh-ed25519(.*)wetty-keyfile$/d' -i ~/.ssh/authorized_keys
+~~~
+
+### Install wetty
+
+**Note:** we are using `-g` for `npm` or `global` for `yarn` along with `--prefix ~/` so that the application symbolic link is installed to our `~/bin` directory and available in our local user's `PATH`.
+
+As your local user run these commands to install `wetty` and `forever`. We will need `forever` later to run wetty in the background.
+
+First, we need to make sure the local user's `~/bin` folder exists and is in the `PATH` for the following commands to work.
+
+~~~bash
+mkdir -p ~/bin && source ~/.profile
+~~~
+
+Please use either the `npm` or `yarn` method and not both. The `yarn` method is recommended but I provide both as you may have a personal preference. The outcome is effectively the same.
+
+`npm `  - optional - use `npm` to install wetty
+
+~~~bash
+npm install -g wetty forever --prefix ~/
+~~~
+
+`yarn` -  recommended - use `yarn` to install wetty
+
+~~~bash
+npm install -g yarn --prefix ~/
+yarn global add wetty forever --prefix ~/
+~~~
+
+Once successfully installed the application should be available in your local user's `PATH`. To test the installation was successful please use this command:
+
+~~~bash
+wetty -h
+~~~
+
+### Accessing the web interface.
+
+This needs to be done here because it is not easy to do in the next steps if `wetty` is running in the terminal.
+
+This command will generate the correct URL you need to visit after using the start up commands in the following section.
+
+~~~bash
+echo https://$(curl -s4 icanhazip.com):3000
```

Comment:
what is this url?

Reply:
This will generate the URL they need to visit in the browser. it assumes they are using an IP.

It's a convenience command but not required in regards to wetty.